src/screens/OrganizationFunds/OrganizationFunds.spec.tsx

```
@@ -51,20 +50,18 @@ const renderOrganizationFunds = (link: ApolloLink): RenderResult => {
                   element={<OrganizationFunds />}
                 />
                 <Route
-                  path="/orgfundcampaign/:orgId/:fundId"
-                  element={
-                    <div data-testid="campaignScreen">Campaign Screen</div>
-                  }
+                  path="/orgfundcampaign/orgId/fundId"
+                  element={<div data-testid="campaignScreen"></div>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)